finagle-thrift/src/main/thrift/tracing.thrift

```diff
@@ -61,12 +167,19 @@ struct BinaryAnnotation {
   4: optional Endpoint host
 }
 
+/*
+ * A trace is a series of spans (often RPC calls) which form a latency graph.
```

Comment:
because only one parent can be specified, this is necessarily a tree, right?  maybe more clear to specify a latency tree.